pact-time.cabal

```diff
@@ -70,20 +84,19 @@ library
         , microlens >=0.4
         , text >=1.2
         , vector >=0.12
-        , vector-space >=0.10
 
-  if flag(with-time)
-    cpp-options: -DWITH_TIME=1
-    other-modules: Pact.Time.Format.External
-    build-depends: time >= 1.8
-  else
-    cpp-options: -DWITH_TIME=0
-    other-modules:
-        Pact.Time.Format.Internal
-        Pact.Time.Format.Locale
-    build-depends:
-        , clock >= 0.7.2
-        , mtl >=2.2
+    if flag(with-time)
+        cpp-options: -DWITH_TIME=1
+        other-modules: Pact.Time.Format.External
+        build-depends: time >= 1.8
+    else
+        cpp-options: -DWITH_TIME=0
+        other-modules:
+            Pact.Time.Format.Internal
+            Pact.Time.Format.Locale
+        build-depends:
+            , clock >= 0.7.2
+            , mtl >=2.2
```

Comment:
This fixes a pre-existing indentation error. The new cabal versions warns about this.